examples/br/backup-nfs.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: pingcap.com/v1alpha1
```

Comment:
What about move the file to `examples/backup` since we may move other examples from `manifests` to this dir later.